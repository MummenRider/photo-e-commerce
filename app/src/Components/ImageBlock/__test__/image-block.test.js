import React from "react";
import { render } from "@testing-library/react";
import LoadImage from "../image-block";

describe("<ImageBlock />", () => {
  it("should render <ImageBlock /> with the passed data", () => {
    const { container, getByAltText } = render(
      <LoadImage
        src={require("Assets/Images/stp_1.jpg").default}
        fallback={require("Assets/Images/stp_1.webp").default}
        alt={"Some kind of picture"}
      />
    );

    expect(getByAltText("Some kind of picture")).toBeTruthy();
  });
});
