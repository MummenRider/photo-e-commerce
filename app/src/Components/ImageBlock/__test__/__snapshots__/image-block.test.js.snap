// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImageBlock /> should render <ImageBlock /> with the passed data 1`] = `
<picture
  class="sc-bdnxRM"
>
  <source
    class="sc-gtsrHT"
    type="image/webp"
  />
  <source
    class="sc-gtsrHT"
  />
  <img
    alt="Some kind of picture"
    class="sc-dlnjwi nsrng"
  />
</picture>
`;
